public/css/footer.css

```diff
@@ -0,0 +1,105 @@
+/***************** Footer Style ****************/
+
+footer {
+    background-color: #575757;
+    width: 100%;
+    height: 150px;
+    display: flex;
+    /* align-content: center; */
+  }
+
+  footer .container {
+    width: 90%;
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  footer .container .logo-container {
+    width: fit-content;
+    margin-left: 150px;
+  }
+
+  footer .container .logo-container span {
+    display: block;
+    color: #FFF;
+  }
+
+  footer .container .logo-container .first {
+    font-size: 34px;
+  }
+
+  footer .container .logo-container .second {
+    font-size: 20px;
+    font-weight: lighter;
+    margin-left: 50px;
+  }
+
+  .container nav ul {
+    display: flex;
+    height: 30%;
+    align-content: center;
+  }
+
+  .container nav ul li {
+    list-style: none;
+    margin-left: 20px;
+  }
+
+  .container nav ul li a {
+    text-decoration: none;
+    transition-duration: .3s;
+  }
+
+  .container nav ul li a .instgram {
```

Comment:
from lines 55 - 64 
why using two selectors for the same element, also you have class for that element so you don't need to use the selector in line 55